core-it/src/test/java/apoc/core/it/LoadCoreEnterpriseTest.java

```diff
@@ -210,6 +217,23 @@ public void testLoadShouldPreventZipBombAttackUtilDecompress() {
         });
     }
 
+    @Test
+    public void testLoadWorksWithIncreasedCompressionRatio() {
```

Comment:
This is a good test, but I think it would be nice with an additional test, that gets zipbomb issue despite not using the default ratio. E.g. if you change the ratio to 101 instead of 100 it should still fail but with an error message that say 101.